src/openms/source/TRANSFORMATIONS/FEATUREFINDER/MultiplexDeltaMassesGenerator.cpp

```diff
@@ -235,10 +237,34 @@ namespace OpenMS
       }
 
     }
+    else if (Leu)
+    {
+      // Leu
+      // We assume each sample to be labelled only once. Hence, we only consider samples_labels_[...][0] below.
```

Comment:
comment could be improved. It's not clear to me why and what the implications are.